helm/botkube/values.yaml

```diff
@@ -6,7 +6,6 @@ replicaCount: 1
 
 image:
   repository: infracloud/botkube
-  tag: "0.6"
```

Comment:
How we are deciding image version after removal of tag ?

Reply:
We'll use chart version as an image tag
```
          image: "{{ .Values.image.repository }}:{{ .Chart.Version }}"
```